src/lib/constants.ts

```diff
@@ -8,6 +8,7 @@ export enum Dependencies {
     ADDRESS = 'dependency:address',
     PAYMENT_METHODS = 'dependency:paymentMethods',
     ORGANIZATION = 'dependency:organization',
+    MEMBERS = 'dependency:members',
```

Comment:
Do we need to make sure to invalidate this when adding a member too?

Reply:
Fixed :)